### DISCUSSION TITLE
Plans for DockManager Control

### DISCUSSION THREAD

**Discussion by shaggygi** [AvaloniaUI/Avalonia#7710] [Q&A] [answered]
Recently came across Avalonia (looks impressive) and was wondering if there are plans to add a DockManager control similar to AvalonDock (in WPF Extended Toolkit)?

I came across [UniDock](https://www.codeproject.com/Articles/5316702/UniDock-A-New-Multiplatform-UI-Docking-Framework-U), but didn't know if there would be something similar out-of-box.

Good job team 👍.  Thx

**Comment by wieslawsoltes** [comment, marked as answer]
There is laso https://github.com/wieslawsoltes/Dock

**Reply by shaggygi** [reply]
@wieslawsoltes Very interesting.  Thx for the link @wieslawsoltes
